update sponsor info (#624)
EuroPython/website: src/content/pages/sponsor/information.mdx

  - **This is the [sample PIPL](https://docs.google.com/spreadsheets/d/1gMajRAPFcdZ2V0Nsw-KKUZuMqGOa9JIs/view) provided (for NON-EU goods)**
  - **Please contact DB Schenker for more Details**

- [Shipping Quotes (.doc)](https://docs.google.com/document/d/1YLOdAEh409HeGeOojxjaNDYX5vZPqZvh)
  - **Contact DB Schenker directly to discuss pricing and conditions**

**Deadlines**

> - Deadline for reciving shipments is **1st of July**.

Reviewer: Maybe we can explicitly say that DB Schenker should receive goods by 1st of July. What do you think?
```suggestion
- DB Schenker should receive shipments by **1st of July**.
```

> - Deadlines are also specified in [Shipping Guidelines & Address (.doc)](https://docs.google.com/document/d/1vtydTQNkLDp2e4NrMg0VIhc2AYM8zMOK/view).

Reviewer: Maybe we can just say that in this doc they could find all the delivery information?

```suggestion
- You could find the complete list of delivery information in the  [Shipping Guidelines & Address (.doc)](https://docs.google.com/document/d/1vtydTQNkLDp2e4NrMg0VIhc2AYM8zMOK/view).
```
